packages/base/any/kernels/5.10-lts/patches/0024-Added-COMPHY-support-for-2500-Base-T-Ethernet.patch

```diff
@@ -0,0 +1,124 @@
+From 82ee941a7b6df5e0b89685845a7f7cf8f04b4405 Mon Sep 17 00:00:00 2001
+From: Elad Nachman <enachman@marvell.com>
+Date: Wed, 21 Jul 2021 17:38:54 +0300
+Subject: [PATCH] Added COMPHY support for 2500 Base-T Ethernet
```

Comment:
Please follow https://chris.beams.io/posts/git-commit/ for writing commit messages.

Reply:
Hopefully new commit 845adc69221e01f9bb1350dfb90c2a80ba815385 is more inline with the link you have provided...